functions.php

```diff
@@ -52,28 +52,22 @@ function sendData($db, $getSeason, $getPremier, $getRunnersUp, $getScore): void
 function getTeamData(array $finals): string 
 {
     $team = '';
-    foreach($finals as $final) {
-        if ($final === []) {
-            return 'Got data?';
-        } else {
-            if($final['Delete'] == '0') {
-                $team .= '<div class="winners-box">' .
-                        '<img src="'.$final['Image'].'" alt="Grandfinal team photo" class="winners-image">' . '<div class="match-stats"><h6 class="team-name">' . $final['Premier'] . '</h6><p>' . $final['Season'] . '</p><p>' . $final['Runner-Up'] . '</p><p>' . $final['Score'] . '</p></div>' . '<form action="delete.php" method="POST"><input type="submit" value="Delete"/>' . '<form class="edit" action="edit.php" method="POST"><input type="submit" value="Edit"/><input class="hidden" name="dataID" value="' . $final['id'] . '"></form></div>';
-            }   
-        }
-    } 
+    foreach($finals as $final) 
+        {
```

Comment:
this bracket goes on the line above, only drop the bracket for functions. so should look like:
```php
foreach($finals as $final)  {
    // code
}
```

Reply:
curly bracket moved up